backend/api/blogpost/fixtures/blogpost.json

```diff
@@ -24,7 +24,7 @@
     "pk": 3,
     "fields": {
       "media_url": "http://www.sample.com",
-      "author_id": 3,
+      "author_id": 1,
```

Comment:
Looks good in general.  I just wanted to ask here--supposed to be `author_id` and not `author`?  I recall Phan had changed `author_id` to `author` in the past

Reply:
I just tried it. It looks like both work. It may be some django magic.